openrpc.json

```diff
@@ -29,6 +29,55 @@
 				}
 			}
 		},
+		{
+			"name": "listServices",
+			"params": [
+				{
+					"name": "filter",
+					"schema": {
+						"type": "string",
+						"enum": [
+							"all",
+							"running",
+							"available",
+							"installed"
+						]
+					},
+					"required": true
+				}
+			],
+			"result": {
+				"name": "listedServices",
+				"schema": {
+					"type": "array",
```

Comment:
should add 

```
"title": "listedServices"
```

so that the generated types get better names

Reply:
I would maybe after this commit lands, then I can go back and update all the stuff for good names, if we don't hit issues like before.